### ISSUE TITLE
A11y_VSCode_Output panel_KeyBoard:Tooltip not visible when keyboard focus moves to the  clear output, toggle scroll lock, close  button

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#76688]
**Environment Details:**
VSCode Version : 1.36.0
OS Version : Win10
Additional Details:
MAS Violated : MAS 1.1.1, MAS 2.1.1

**Repro Steps:**
1) Open Visual studio code editor  
2) Press ctrl+shift+u
3) Navigate to the clear output, toggle scroll lock, close  button using keyboard
4) Observe the tool tip is visible or not when the focus is on the button. 

**Actual:**
Tool tip is not visible when focus moves to the clear output, toggle scroll lock, close  button.

**Expected:**
Tool tip should be visible when focus moves to the keyboard

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Keyboard  users will face dificulty to understand the control.

**MAS Reference**:
MAS 1.1.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

MAS 2.1.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment For Reference:** 
[A11y_VSCode_Output panel_ScreenReader_ Tooltip not visible.pptx](https://github.com/microsoft/vscode/files/3362051/A11y_VSCode_Output.panel_ScreenReader_.Tooltip.not.visible.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of all the issues the bot detected
Also dup of https://github.com/microsoft/vscode/issues/52016
